backend/curriculum_tracking/serializers.py

```diff
@@ -707,3 +707,16 @@ def save(self,**kwargs):
         instance.set_flavours(flavour_names)
         return instance
 
+class CurriculumContentRequirementSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.CurriculumContentRequirement
+        fields = ["id", "curriculum","content_item"]
+
+    curriculum = serializers.SerializerMethodField("get_curriculum_name")
+    content_item = serializers.SerializerMethodField("get_content_item_title")
+
+    def get_curriculum_name(self, instance):
+        return instance.curriculum.name
+
+    def get_content_item_title(self, instance):
+        return instance.content_item.title
```

Comment:
Unless I'm missing something, this seems like it only returns one content_item title at a time? It would be good to get a list of all content items (both title and id) associated with each curriculum

Reply:
I realise I was thinking about it wrong, and it should return all the content items. But the content_id part is needed.
I'm trying to run the tests and hitting some errors but not sure if the problem is on my side...

Reply:
me again.. :) ok so got things working and ran the development server, seems we can see the id of the content items, but not the id of the curriculum (only the name of the curriculum), is it possible to see the id of the curriculum too so we can filter by a curriculum and see all the content items related to it?

Reply:
So here is screenshot of your new api from me running the dev server:
<img width="773" alt="Screenshot 2022-08-01 at 16 30 23" src="https://user-images.githubusercontent.com/50909846/182173092-60d0ea11-4212-41fa-9165-144871165d8a.png">

but like this shot of the agile card api, be good to see content item id (and by the same token curriculum item id:
<img width="1009" alt="Screenshot 2022-08-01 at 16 26 55" src="https://user-images.githubusercontent.com/50909846/182173481-edcb56a4-b293-480f-aafa-080024bd029c.png">

Does this make sense?